webrtc.html

```diff
@@ -11786,6 +11809,16 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
           the <code><a>RTCError</a></code> constructor is 0.</p>
         </section>
         <section>
+          <h2>RTCError.prototype.receivedAlert</h2>
+          <p>The initial value of the <code>receivedAlert</code> property of the prototype for 
+          the <code><a>RTCError</a></code> constructor is null.</p>
```

Comment:
What's the type of these properties? Are they strings (`"bad_certificate"`) or integers (`42`)? Might also want to point to an RFC where they're defined.